draft-ietf-tls-dtls-connection-id.md

```diff
@@ -535,13 +535,21 @@ The topic of peer address updates is discussed in {{peer-address-update}}.
 
 #  IANA Considerations
 
-IANA is requested to allocate an entry to the existing TLS "ExtensionType
-Values" registry, defined in {{RFC5246}}, for connection_id(TBD1) as described
-in the table below. IANA is requested to add an extra column to the
-TLS ExtensionType Values registry to indicate whether an extension is only
+This document requests three actions by IANA. 
+
+## Extra Column to TLS ExtensionType Values Registry
+
+IANA is requested to add an extra column to the
+"TLS ExtensionType Values" registry to indicate whether an extension is only
 applicable to DTLS and to include this document as an additional reference
 for the registry.
```

Comment:
We might give IANA some guidance on what name to use for the column and what value(s) should appear in it.